app/models/zone.rb

```diff
@@ -35,6 +36,8 @@ class Zone < ApplicationRecord
   include AggregationMixin
   include ConfigurationManagementMixin
 
+  scope :visible, -> { where(:visible => true) }
+
```

Comment:
I thought you were removing the visible column?

Reply:
I removed it as `default_scope`, but cloud EMS cannot have nil zone (validation), so I need to have some different zone, which can be filtered by UI, that's why kept visible column and add MaintenanceZone to Zone.seed